src/components/LoginForm.jsx

```diff
@@ -120,13 +150,37 @@ function LoginForm() {
       passwordVerificationError.status ||
       passwordError.status
     ) {
-      console.log("Can't save");
       return;
     }
 
-    // Handle api call here
-    console.log("save");
-    navigate("/dashboard");
+    try {
+      const response = await createAccount(email, password);
+      alert("Created Account");
+      navigate("/dashboard");
```

Comment:
After a user has successfully signed up at `/signup/` they're not logged in, so I think this should redirect to the login form (maybe there could be a message to the effect that your account has been created, and now you need to log in with your credentials).

Alternatively, the client could silently make another request to `/login/` with the  user's new credentials before redirecting to the dashboard.

Reply:
Yea this is a good call, I will make it so that the users will login themselves in to confirm account is created